lib/Echidna/Types/Tx.hs

```diff
@@ -116,3 +116,7 @@ getResult (VMFailure PrecompileFailure)         = ErrorPrecompileFailure
 getResult (VMFailure UnexpectedSymbolicArg)     = ErrorUnexpectedSymbolic
 getResult (VMFailure DeadPath)                  = ErrorDeadPath
 getResult (VMFailure (Choose _))                = ErrorChoose -- not entirely sure what this is
+
+makeSingleTx :: Addr -> Addr -> W256 -> TxCall -> [Tx]
```

Comment:
you can use `Echidna.Types.Signature.SolCall` instead of `TxCall` and there won't be need for `error`